gta3/gta3.json

```diff
@@ -29879,6 +29879,35 @@
             "1.0"
           ]
         },
+        {
+          "output": [
+            {
+              "name": "width",
+              "type": "float",
+              "source": "var_any"
+            },
+            {
+              "name": "height",
+              "type": "float",
+              "source": "var_any"
+            }
+          ],
+          "id": "0C1B",
+          "name": "IMGUI_GET_DISPLAY_SIZE",
+          "num_params": 0,
```

Comment:
num_params is the sum of input and output params. should be 2

Reply:
opps, fixing..

Reply:
please bump the version in the meta section and in version.txt. also update timestamp if you can


if you edit opcodes via the web editor you can add the command once and then clone it to other games, all changes are synchronized.